src/terminal/char_grid/cursor.rs

```diff
@@ -24,10 +24,20 @@ pub struct Cursor {
     pub coords: Coords,
     pub style: Styles,
     pub text_style: Styles,
+    config: Config,
```

Comment:
@withoutboats what do you think about storing a reference to a `Config` instead?